pkg/network/usm/ebpf_gotls.go

```diff
@@ -602,6 +602,11 @@ func (p *GoTLSProgram) unhookBinary(bin *runningBinary) {
 
 func (p *GoTLSProgram) detachHooks(probeIDs []manager.ProbeIdentificationPair) {
 	for _, probeID := range probeIDs {
+		probe, ok := p.manager.GetProbe(probeID)
+		if !ok {
+			continue
+		}
+		ebpfcheck.RemoveProgramNameMapping(probe.ID())
```

Comment:
Reviewing how GoTLS works, I don't think this is correct because there is only one instance of the ebpf program loaded correct? It is just attached multiple times to different binaries?

The `ebpfcheck` is only aware of attached programs, so I think they all share the same program ID.